recipes/sweets/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "sweets" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/sweets/archive/v{{ version }}.tar.gz
+  sha256: e62aa647d7aab8185e92c28cb4611d00cf4f62071b5ff27d48cbde614cc62ace
+
+build:
+  entry_points:
+    - sweets = sweets.cli:main
+  noarch: python
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=64.0
+    - setuptools-scm >=6.2
+    - pip >=21.3
+  run:
+    - python >=3.8
+    - pip >=21.3  # https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#editable-installation
+    - git  # for pip install, due to setuptools_scm
```

Comment:
Are you sure you need these two at run-time? Your comments seems to be for installation, not running.

```suggestion
```

Reply:
ah thank you! I missed that in this one.